fix(ui): improve navbar for smaller screens

ParthJohri opened the pull request:
<!-- Thanks for creating this pull request 🤗 Please make sure you followed the conventional commit -->
# Fix: Navbar Fix For Mobile Version [Issue No: 866](https://github.com/glasskube/glasskube/issues/866)

<!-- If this pull request closes an issue, please mention the issue number below -->
Closes #866 <!-- Issue # here -->

## 📑 Changes

 - Used the bootstrap class [`d-flex flex-row`](https://getbootstrap.com/docs/4.0/utilities/flex/) to bring down all the elements properly in a row. Since `ul` element by default had `flex-column` due to which they were stacking over each other.

## ✅ What type of PR?
- [x] ✨ Enhancement
- [x] ✨ UI Change

## ℹ Screen Recording

https://github.com/glasskube/glasskube/assets/76129377/f02ece71-4a06-416a-b188-5edc495d796e

christophenne reviewed (commented):
Thanks a bunch @ParthJohri!

Looks good to me, the only very minor thing I'd like to mention, is that on the smaller screen, it would be nice if the navbar contents don't get centered, i.e. the glasskube logo and the links should stay left, and the Cloud link and Github button on the right – could you add that to the PR please?

Granted, it was a bit difficult for you to see – I see that you probably have no kubernetes cluster to test this with (if you are curious, you can install minikube locally and test against that, to see the UI in full action), and therefore you didn't get to the packages page yet. But on the support page that you see, there is also this blue "Glasskube Cloud signup" banner below the navbar. I think the navbar contents should be as wide as this banner, than it should also look better on the packages page. 

For reference, this is what it looks like now, on the packages page:
![Screenshot from 2024-06-26 10-52-16](https://github.com/glasskube/glasskube/assets/7556574/21744a2b-2a54-4c47-8de8-3dcfe7f9a50d)

christophenne commented:
Oh and two more things:
* I think it's okay now without the mobile menu
* please run `make lint-fix` and then commit your changes – this will fix the build :)

ParthJohri commented:
Thanks a lot @christophenne for all the suggestions, sure I would put glasskube logo and the links on the left, and the Cloud link and Github button on the right and I will install minikube to see the full ui, if I encounter any doubts, I will reach out to you, thank you!

ParthJohri commented:
@christophenne I apologise but the build is failing as the command `make lint-fix` is throwing me errors, so had to commit without using it. I had a look at the Makefile and am trying to debug as to why is it appearing, I will also share the query on the discord group as to how to solve it. 

However, I have resolved the suggestions and have attached the screenshots for the navbar as well:-

If I missed anything, please let me know, I will complete the needful.

## Changes 

1. Added an inner container to distribute Glasskube logo and links on the left side, and another container for the CloudLink and GitHub.
2. Updated the container size to large

## Screenshots
![navbar-glasskube1](https://github.com/glasskube/glasskube/assets/76129377/65417c5e-0ff2-48ed-ae8f-a7da1072bbc0)
![navbar-glasskube2](https://github.com/glasskube/glasskube/assets/76129377/4dde2cb1-62a8-4df0-a1a6-11b43125a93f)
![navbar-glasskube3](https://github.com/glasskube/glasskube/assets/76129377/dd375970-5966-4155-b06a-f0ea4c03c82e)

christophenne reviewed (approved):
LGTM, thanks!
